frontends/web/src/routes/bitboxbase/bitboxbase.tsx

```diff
@@ -70,35 +85,46 @@ class BitBoxBase extends Component<Props> {
         {
             t,
             ip,
-            bitboxBaseIDs,
+            detectedBases,
         }: RenderableProps<Props>,
         ) {
             return (
             <div class="contentWithGuide">
                 <div class="container">
-                    <Header title={t('bitboxBase.title')} />
+                    <Header title={<h2>{t('bitboxBase.title')}</h2>} />
                     <div class="innerContainer scrollableContainer">
                         <div class="content padded">
+                            {
+                                detectedBases ?
```

Comment:
Could you always display both? This makes it a bit annoying to test when I have the base middleware running locally and have some bases reachable with mdns at the same time.

Reply:
Done.